x-pack/plugins/uptime/server/graphql/monitors/schema.gql.ts

```diff
@@ -65,10 +71,33 @@ export const monitorsSchema = gql`
     y: Int
   }
 
+  "Represents a monitor's duration performance in microseconds at a point in time."
+  type MonitorDurationAreaPoint {
+    "The timeseries value for this point in time."
+    x: UnsignedInteger!
+    "The min duration value in microseconds at this time."
+    y0: Float
+    "The max duration value in microseconds at this point."
+    y: Float
+  }
+
+  "Represents the average monitor duration ms at a point in time."
+  type MonitorDurationAveragePoint {
```

Comment:
I feel like at some later point we'll have more generic `CartesianPoint` types, but I'm fine with this for now.

Reply:
I tend to think of these chart-related bits of code as in a state of flux that will be overwritten/deleted (because we're going to need to re-implement the charts when new EUI components are available). We will want to take care that we approach it with nuance and sensibility when we overwrite what we have today.